### PR TITLE
Bug/naming convention for isOnWishlist and IsOnCart

### LINKED ISSUE #5592: Refactor naming convention for `isOnCart` and  `isOnWishlist`
## Current behavior

Both `isOnCart` and  `isOnWishlist` are not gramatically correct names. 

## Expected behavior

The appriopriate names for these should be `isInCart` and  `isInWishlist`. All occurrences should be fixed.

## Version of Vue Storefront

* [ ] Vue Storefront
* [x] Vue Storefront Next

## Can you handle fixing this bug by yourself?
* [x] YES
* [ ] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer. (doesn't apply to Next)
Pick one option.

* [x] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
* [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
* [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details

* Browser: 
* OS: 
* Node: 
* Code Version:

### PR DESCRIPTION
**Pull request by Baroshem**
### Related Issues

Closes #5592 

### Short Description and Why It's Useful

A change in the naming convention from isOnWishlist and IsOnCart to isInWishlist and IsInCart.
Related Storefront UI issue -> https://github.com/vuestorefront/storefront-ui/issues/1718

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

* [x] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
* [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
* [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
* [] No upgrade steps required (100% backward compatibility and no breaking changes)
* [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature
* [x] (Next only) I've followed this [instruction](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and I've created a `.js` file with information about my Pull Request

**IMPORTANT NOTICE**

* Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

* [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/core/docs/migrate/projects-2.2.0.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ We have changes a bit the naming and signatures of composable functions. Below i
 |      useWishlist      |   removeFromWishlist     |       (product: WISHLIST_ITEM, customQuery?: CustomQuery)       |       ({ product: WISHLIST_ITEM, customQuery?: CustomQuery })        |
 |      useWishlist      |   load     |       (customQuery?: CustomQuery)       |       ({ customQuery?: CustomQuery } = {})        |
 |      useWishlist      |   clearWishlist     |       ()       |       ()        |
-|      useWishlist      |   isOnWishlist     |       (product: PRODUCT)       |       ({ product: PRODUCT })        |
\ No newline at end of file
+|      useWishlist      |   isInWishlist     |       (product: PRODUCT)       |       ({ product: PRODUCT })        |
```
Comment (filipsobol) [inline packages/core/docs/migrate/projects-2.2.0.md:45]: This file shouldn't be modified, as it's a migration guide for previous version.

**Review by filipsobol — CHANGES_REQUESTED** [review]

File: packages/core/docs/migrate/projects-2.2.0.md (hunk 1 above)
Comment (Baroshem) [reply packages/core/docs/migrate/projects-2.2.0.md]: Thanks @filipsobol. I removed this change in the latest commit

File: packages/core/docs/changelog/5592.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  description: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist',
+  link: 'https://github.com/vuestorefront/vue-storefront/issues/5592',
+  isBreaking: true,
+  breakingChanges: [
+    {
+      module: 'commercetools',
```
Comment (filrak) [inline packages/core/docs/changelog/5592.js:7]:
```suggestion
      module: 'useCart/useWishlist',
```

File: packages/core/docs/changelog/5592.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      before: 'variables and methods had names isOnCart and isOnWishlist',
+      after: 'variables and methods have names isInCart and isInWishlist',
+      comment: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist fro commercetools'
```
Comment (filrak) [inline packages/core/docs/changelog/5592.js:10]:
```suggestion
      comment: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist'
```

File: packages/core/docs/changelog/5592.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    {
+      module: 'core',
+      before: 'variables and methods had names isOnCart and isOnWishlist',
+      after: 'variables and methods have names isInCart and isInWishlist',
+      comment: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist for core'
+    }
```
Comment (filrak) [inline packages/core/docs/changelog/5592.js:17]:
```suggestion

```

File: packages/core/docs/changelog/5592.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  description: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist',
+  link: 'https://github.com/vuestorefront/vue-storefront/issues/5592',
+  isBreaking: true,
+  breakingChanges: [
+    {
+      module: 'commercetools',
+      before: 'variables and methods had names isOnCart and isOnWishlist',
+      after: 'variables and methods have names isInCart and isInWishlist',
+      comment: 'fix naming convention for isOnCart and isOnWishlist to isInCart and isInWishlist fro commercetools'
+    },
```
Comment (filrak) [inline packages/core/docs/changelog/5592.js:11]:
```suggestion
    }
```

**Review by filrak — APPROVED** [review]

**Pull request merged**